pinkopy/base_session.py

```diff
@@ -137,7 +137,7 @@ def request(self, method, path, attempt=None, headers=None, payload=None,
             else:
                 log.info('CMDBSession made request to {0}'.format(url))
                 return res
-        except requests.exceptions.HTTPError as err:
+        except requests.HTTPError as err:
```

Comment:
Oh I see. That's interesting. I wonder if it is based on your Python or requests versions. Whatever the case, keep it then. I didn't know that. I only tried it in Python 2.7 and 3.5 with requests 2.9.1.